mllib/src/main/scala/org/apache/spark/ml/tree/Node.scala

```diff
@@ -287,6 +292,41 @@ private[tree] class LearningNode(
     }
   }
 
+  /**
+   * Method testing whether a node is a leaf.
+   * @return true iff a node is a leaf.
+   */
+  private def isLeafNode(): Boolean = leftChild.isEmpty && rightChild.isEmpty
+
+  /** True iff the node should be a leaf. */
```

Comment:
Likewise an example of the kind of comment that seems to just restate the method name.